src/amber/router/parsers/form_data.cr

```diff
@@ -0,0 +1,19 @@
+module Parsers
```

Comment:
All of these parsers should be nested under the `Amber::Router::Parser` module instead of just `Parsers` to minimize leakage outside of the namespace.

Reply:
Yes missed that. Good observation